integrationtest/initial_release/shellHelper.d

```diff
@@ -18,25 +18,45 @@ module integrationtest.initial_release.shellHelper;
 
     Params:
         wd = directory to run the command in
-        command = command to run
+        command = command to run and arguments to pass to it
 
     Returns:
         output of the command
 
 *******************************************************************************/
 
-string cmd ( string wd, string command )
+string cmd ( string wd, const(string[]) command )
 {
-    import std.process : executeShell, Config;
+    import std.process : execute, Config;
     import std.string : strip;
 
     string[string] env;
     env["HOME"] = wd;
 
-    auto c = executeShell(command, env, Config.none, size_t.max, wd);
+    auto c = execute(command, env, Config.none, size_t.max, wd);
 
     if (c.status != 0)
         throw new Exception(c.output);
 
     return strip(c.output);
 }
+
+/*******************************************************************************
+
+    Runs a command and returns the output
+
+    Params:
+        wd = directory to run the command in
+        command = command to run (including arguments). Arguments will be split
+                  based on spaces
+
+    Returns:
+        output of the command
+
+*******************************************************************************/
+
+string cmd ( string wd, string command )
```

Comment:
This isn't really the same as executing in shell (as it may split args differently than `split` call). Intentional?

Reply:
Nope, it was just for convenience for the tests or for commands that don't have user input, but splitting arguments doesn't trigger expansions (as a shell will do, like `*.md` or `$HOME`) and won't accept shell operators (like `|` or `;`, etc.). All these are the source to errors / security holes.

Reply:
Oh, I read your question wrongly. Yes, it is intended :)

It is documented that it just does a dump split.